openapi_operations.go

```diff
@@ -82,6 +82,33 @@ func (r Route[ResponseBody, RequestBody]) AddTags(tags ...string) Route[Response
 	return r
 }
 
+// AddError adds an error to the route.
+func (r Route[ResponseBody, RequestBody]) AddError(code int, description string, errorType ...any) Route[ResponseBody, RequestBody] {
+	addResponse(r.mainRouter, r.Operation, code, description, errorType...)
+	return r
+}
+
+func addResponse(s *Server, operation *openapi3.Operation, code int, description string, errorType ...any) {
+	responseSchema := schemaTagFromType(s, HTTPError{})
+	if len(errorType) > 0 {
+		responseSchema = schemaTagFromType(s, errorType[0])
+	}
```

Comment:
Small optimization. We can either use an `if...else` or wrap this in a function to avoid the `if...else`, would avoid a call to `schemaTagFromType`. Anywho just a nitpick - this would be pretty quick anyway.

Reply:
Done! Thank you